plugin.php

```diff
@@ -573,12 +577,25 @@ function rest_handle_options_request( $response, $handler, $request ) {
 
 		foreach ( $endpoints as $endpoint ) {
 			$accept = array_merge( $accept, $endpoint['methods'] );
+
+			$methods = array_keys( $endpoint['methods'] );
+			$method = array_shift( $methods );
```

Comment:
Mainly this was so it's a constant-time lookup internally in `dispatch`. Endpoints often support more than one method as well, all EDITABLE supports POST, PUT, and PATCH.